### ISSUE TITLE
Kick the ball

### ISSUE THREAD

**Issue by Adman** [RoboCupJuniorTC/webots-soccer-sim-playground#13]
Let's see if it is possible that the robot sends some "kick" signal to supervisor and supervisor sets the ball's velocity appropriately.

**Comment by mrshu** [comment]
@Adman a year later, and here we are :slightly_smiling_face: 

Would you still be up for having/doing something like this?

**Comment by Adman** [comment]
@mrshu yeah, I can give it a try once more important things like tests/sensors are done.
However, I'm not sure whether we can push this to this year's competitions as it requires quite a lot of testing.
